src/components/trade/PriceSuggestions/PriceSuggestions.tsx

```diff
@@ -118,7 +117,9 @@ export interface Props {
   // Market
   baseToken: TokenDetails
   quoteToken: TokenDetails
+  sellToken: TokenDetails
   isPriceInverted: boolean
+  wasPriorityAdjusted: boolean
```

Comment:
Please, let's not propagate `wasPriorityAdjusted` this prop down all the components. you just let the component know what's your selection of base/quote and if it is inverted or not.